ddtrace/internal/constants.py

```diff
@@ -18,6 +18,8 @@
 DEFAULT_SERVICE_NAME = "unnamed_python_service"
 # Used to set the name of an integration on a span
 COMPONENT = "component"
+_HIGHER_ORDER_TRACE_ID_BITS = "_dd.p.tid"
```

Comment:
```suggestion
HIGHER_ORDER_TRACE_ID_BITS_KEY = "_dd.p.tid"
```

doesn't have to be internal since the module is internal